Utilities/General/ibeos/das_client

```diff
@@ -23,18 +23,18 @@ HELP=
 for ((i=1; i<=$#; i++)); do
   next=$((i+1))
   case  ${!i} in
-    --query=*  ) QUERY=$(echo ${!i} | sed 's|.*=||') ;;
-    --query    ) QUERY=${!next} ;;
-    --format=* ) FORMAT=$(echo ${!i} | sed 's|.*=||') ;;
-    --format   ) FORMAT=${!next} ;;
-    -h|--help  ) HELP=YES ;;
+    --query=* | -query=* ) QUERY="${!i#*=}" ;;
```

Comment:
In unix command lines, double dashes are used for full word arguments while single dashes are only supposed to be single letter. Why do you want to break that convention here?

Reply:
The convention was already broken by `dasgoclient`, unfortunately. I just want to provide compatibility between the two tools (which are intended to be compatible).